packages/portal/swagger-ui-web-component/src/styles/overrides.css

```diff
@@ -6,3 +6,23 @@
 .swagger-ui .wrapper {
   padding-top: 0;
 }
+
+.swagger-ui .opblock {
+  background: none !important;
+}
+
+.swagger-ui .json-schema-2020-12-accordion,
+.swagger-ui .json-schema-2020-12-expand-deep-button {
+  background-color: var(--kong-ui-section_colors-body) !important;
```

Comment:
question: why a mix of kebab-case and snake_case?

Also, do we not need to provide a fallback value?

Reply:
The var comes from portal, I don't have a say